Sources/_StringProcessing/ConsumerInterface.swift

```diff
@@ -11,6 +11,13 @@
 
 @_implementationOnly import _RegexParser
 
+extension Character {
+  var singleScalarAsciiValue: UInt8? {
```

Comment:
Weakly prefer underscoring this, since it could otherwise be confusable with API. I realize we've not been super consistent on this.